Mathlib/GroupTheory/Perm/Cycle/Concrete.lean

```diff
@@ -287,7 +287,6 @@ theorem SameCycle.toList_isRotated {f : Perm α} {x y : α} (h : SameCycle f x y
   by_cases hx : x ∈ f.support
   · obtain ⟨_ | k, _, hy⟩ := h.exists_pow_eq_of_mem_support hx
     · simp only [coe_one, id, pow_zero] at hy
-      -- Porting note: added `IsRotated.refl`
       simp [hy, IsRotated.refl]
```

Comment:
Is `IsRotated.refl` still needed here? If not, it looks like the porting note still applies.

Reply:
Restored.